common/source/docs/common-custom-firmware.rst

```diff
@@ -41,8 +41,348 @@ To submit a build request, follow these steps:
         :target: ../_images/mission-planner-load-custom-firmware.png
         :width: 450px
 
-custom-beta
------------
+Versions
+========
+
+Latest
+------
+This is the current development branch of ArduPilot. It includes new features, enhancements, clean-ups, etc. but has not been throught a release cycle of beta testing. It has been alpha tested and thoroughly tested in ArduPilot's Continuous Integration auto-test suite. Many experienced users select this to get the latest features but it has the highest risk, in terms of potential bugs.
+
+Stable
+------
+These are fully released versions of the code. There are usually several minor releases for each major release number which correct any anomalies found during the beta testing phases. These releases, have the lowest risk. It is recommended that the highest numbered major/minor release be used in a given release sequence.
+
+Beta
+----
+This is a release of the development branch used for widespread testing and feedback. While the risk of major issues is low, it is expected that minor behavioral deviations may pop up. These are normally addressed in later beta releases of a major new release before declaring the code as a stable release.
+
+Guide to Size Impacts
+=====================
+
+Often, the custom build server is used to create firmware with features that have been removed in the standard firmware to conserve flash space, but desired by the user. If a build fails to complete due to lack of flash, the user may remove features/sensors that he does not need in order to make room for those he wants to add. In order to facilitate those choices, the table below shows what impact a feature's removal has on flash size. Note that a removed feature in this list automatically removed all features dependent upon it.
```

Comment:
"that he does not need" could be replaced with, "that are not needed"

Reply:
reworded..thanks